src/tests/components/Search.test.js

```diff
@@ -22,4 +22,10 @@ describe("Search", () => {
 
         expect(getByRole("textbox", { name: "Search" }).value).toEqual("French");
     });
+
+    it("contains a level parameter for comparing analytics", () => {
```

Comment:
I didn't quite understand this description on first reading. Maybe we could make it slightly clearer?

```suggestion
    it("contains a 'level' query parameter for comparing analytics between this app and Course Finder", () => {
```